apiserver/common/storagecommon/storage_test.go

```diff
@@ -84,7 +82,7 @@ func (s *storageAttachmentInfoSuite) TestStorageAttachmentInfoPersistentDeviceNa
 	s.st.CheckCallNames(c, "StorageInstance", "StorageInstanceVolume", "VolumeAttachment", "BlockDevices")
 	c.Assert(info, jc.DeepEquals, &storage.StorageAttachmentInfo{
 		Kind:     storage.StorageKindBlock,
-		Location: filepath.FromSlash("/dev/sda"),
+		Location: "/dev/sda",
```

Comment:
filepath is generally quite dangerous when you are talking about paths on *another* machine.
We ran into things like this around "bootstrap", etc trying to set /var/lib paths on the Ubuntu machine that is being started.